demisto_sdk/commands/common/hook_validations/docker.py

```diff
@@ -180,10 +174,11 @@ def find_latest_tag_by_date(tags):
         latest_tag_date = datetime.now() - timedelta(days=400000)
         for tag in tags:
             tag_date = datetime.strptime(tag.get('last_updated'), '%Y-%m-%dT%H:%M:%S.%fZ')
-            if tag_date >= latest_tag_date:
+            if tag_date >= latest_tag_date and tag.get('name') != 'latest':
```

Comment:
what happens to a docker image that `latest` is the only tag it got ?

Reply:
latest_tag_name will stay "latest" and we will get the error:
'The docker image tag is labeled "latest" please create a valid docker image tag'